hubmapdbgap/create.py

```diff
@@ -400,12 +400,7 @@ def submission(
     for hubmap_id in tqdm(hubmap_ids):
         commands = f"# {hubmap_id}\n"
         for index, row in dataset.iterrows():
-            if prepend_sample_id:
-                datum.extend(["fastq", f'{hubmap_id}-{row["filename"]}', row["md5"]])
-                commands = f'{commands}cp -v "{dataset_directory}/{row["filename"]}" ./{hubmap_id}-{row["filename"]}\n'
-            else:
-                datum.extend(["fastq", row["filename"], row["md5"]])
-                commands = f'{commands}cp -v "{dataset_directory}/{row["filename"]}" ./{row["filename"]}\n'
+            commands = f'{commands}cp -v "{dataset_directory}/{row["filename"]}" ./{row["filename"]}\n'
```

Comment:
why are you removing this functionality @j-uranic ?

Reply:
As a potential fix to the script.sh output issue.